jxl/src/features/spline.rs

```diff
@@ -59,6 +63,21 @@ pub struct QuantizedSpline {
     pub sigma_dct: [i32; 32],
 }
 
+// Maximum number of spline control points per frame is
```

Comment:
Is this meant to be here?

Reply:
we will use it soon, but right now it is not needed, deleted.